packages/manager/modules/pci/src/projects/new/payment/controller.js

```diff
@@ -1,4 +1,15 @@
+import find from 'lodash/find';
+import {
+  ORDER_FOLLOW_UP_STATUS_ENUM,
+  ORDER_FOLLOW_UP_STEP_ENUM,
+} from '../../projects.constant';
+
 const getPaymentMethodTimeoutLimit = 30000;
+const ANTI_FRAUD = {
+  CASE_FRAUD_REFUSED: '(error 906)',
+  INTERVAL: 2000,
```

Comment:
What interval is this? Probably, a better name would help :)

Reply:
Right, renamed to `POLLING_INTERVAL`